crates/sui/tests/full_node_tests.rs

```diff
@@ -86,6 +86,27 @@ async fn test_full_node_follows_txes() -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+#[sim_test]
+async fn test_full_node_syncs_checkpoints() -> Result<(), anyhow::Error> {
+    let test_cluster = init_cluster_builder_env_aware().build().await?;
+    let sui_node: SuiNode = start_a_fullnode(&test_cluster.swarm, false).await?;
+    let node = if cfg!(msim) {
+        &sui_node
```

Comment:
Why was this necessary? the `SuiNode` api should not change in the simulator.

Reply:
What exactly are you referring to?